routers/user/home.go

```diff
@@ -114,8 +114,12 @@ func Dashboard(ctx *context.Context) {
 	ctx.Data["SearchLimit"] = setting.UI.User.RepoPagingNum
 	// no heatmap access for admins; GetUserHeatmapDataByUser ignores the calling user
 	// so everyone would get the same empty heatmap
-	ctx.Data["EnableHeatmap"] = setting.Service.EnableUserHeatmap && !ctxUser.KeepActivityPrivate
-	ctx.Data["HeatmapUser"] = ctxUser.Name
+	if setting.Service.EnableUserHeatmap && !ctxUser.KeepActivityPrivate {
+		data, err := models.GetUserHeatmapDataByUser(ctxUser)
```

Comment:
This err should probably still be handled.

Reply:
I don't see how, logging?

Reply:
I guess I could hide the placeholder element on error in JS, but not sure what to do here exactly.

Reply:
Similar to the other calls in this func, probably returning a server error. It looks like that func only returns an error if the DB transaction fails.

Reply:
At the very least logging it, but if it just hides a UI element we'll get bugs if it ever fails consistently and the heatmap silently disappears.

Reply:
Added this from JS:

<img width="704" alt="image" src="https://user-images.githubusercontent.com/115237/99573290-3262ad80-29d6-11eb-93f6-2e57291bcd34.png">

Reply:
It should still log it so that we get the full error and stacktrace in the log

Reply:
Question is really, do we want to return a 500 if only this part of the page fails to load? Isn't  that a bit much?

Reply:
Logging it won't inherently return a 500.  
Though, I would argue that since the only error that can happen here is a DB error, it likely indicates a larger issue somewhere.

Reply:
Added the 500, we'll see how it goes.